.github/actions/spelling/allow.txt

```diff
@@ -8,17 +8,21 @@ apiclienttenant
 apikey
 apps
 appsettings
+argv
```

Comment:
Some of these look generic enough for the common configuration, but some very rare. For the latter, rather use `#spell-check-ignore-line`

Reply:
Actually, most of those rare words are simply specific to NE. Are we planning to support project-level dictionaries anytime soon? Because then I'd rather move them into one of those than commenting them everywhere.

Reply:
Yeah, good point, the spelling configuration in most repositories (but definitely in OSOCE) will be completely overhauled in https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/346.
So, you can apply the current practices to make it easier on rolling this out without having to worry about upcoming changes.

See my comment about using per-line ignores here: https://github.com/Lombiq/GitHub-Actions/issues/134#issuecomment-1348137186
If you can expect that a specific word will appear in more places in the future, then putting it in a dictionary is OK. Based on applying this concept to https://github.com/OrchardCMS/OrchardCore.Commerce/pull/250 and some other repos, I'd say that maximum 1 or 2 occurrences of a rare word (if you don't expect more) is OK to ignore.

Reply:
For the record - all errors by the spell checker when no new entries are added:

> `Autoprefix` is not a recognized word. (unrecognized-spelling)
`EBUSY` is not a recognized word. (unrecognized-spelling)
`EOL` is not a recognized word. (unrecognized-spelling)
`EOL` is not a recognized word. (unrecognized-spelling)
`EOL` is not a recognized word. (unrecognized-spelling)
`Lintrc` is not a recognized word. (unrecognized-spelling)
`Lintrc` is not a recognized word. (unrecognized-spelling)
`Promisified` is not a recognized word. (unrecognized-spelling)
`Resx` is not a recognized word. (unrecognized-spelling)
`argv` is not a recognized word. (unrecognized-spelling)
`argv` is not a recognized word. (unrecognized-spelling)
`argv` is not a recognized word. (unrecognized-spelling)
`argv` is not a recognized word. (unrecognized-spelling)
`argv` is not a recognized word. (unrecognized-spelling)
`binstubs` is not a recognized word. (unrecognized-spelling)
`chdir` is not a recognized word. (unrecognized-spelling)
`chdir` is not a recognized word. (unrecognized-spelling)
`chdir` is not a recognized word. (unrecognized-spelling)
`chdir` is not a recognized word. (unrecognized-spelling)
`constitue` is not a recognized word. (unrecognized-spelling)
`copyfiles` is not a recognized word. (unrecognized-spelling)
`copyfiles` is not a recognized word. (unrecognized-spelling)
`copyfiles` is not a recognized word. (unrecognized-spelling)
`copyfiles` is not a recognized word. (unrecognized-spelling)
`copyfiles` is not a recognized word. (unrecognized-spelling)
`errored` is not a recognized word. (unrecognized-spelling)
`lintrc` is not a recognized word. (unrecognized-spelling)
`loglevel` is not a recognized word. (unrecognized-spelling)
`npmrc` is not a recognized word. (check-file-path)
`npmrc` is not a recognized word. (check-file-path)
`npmrc` is not a recognized word. (unrecognized-spelling)
`promisify` is not a recognized word. (unrecognized-spelling)
`promisify` is not a recognized word. (unrecognized-spelling)
`symlink` is not a recognized word. (unrecognized-spelling)
`symlink` is not a recognized word. (unrecognized-spelling)
`symlinked` is not a recognized word. (unrecognized-spelling)
`symlinked` is not a recognized word. (unrecognized-spelling)
`symlinks` is not a recognized word. (unrecognized-spelling)
`textlint` is not a recognized word. (unrecognized-spelling)
`textlint` is not a recognized word. (unrecognized-spelling)
`textlint` is not a recognized word. (unrecognized-spelling)
`thenable` is not a recognized word. (unrecognized-spelling)

Reply:
That's not too bad! Just add them here to allow.txt for now (or ignore in-code whatever you see fit for that) and I'll take care of the rest during the refactoring.

Reply:
I did what Zoltán suggested - ignored the rare ones, re-added the common ones.